src/main/java/io/vavr/control/Option.java

```diff
@@ -456,6 +498,16 @@ public final T getOrElse(T other) {
      * <p>
      * Please note, that the other value is lazily evaluated.
      *
+     * <pre>{@code
+     * Supplier<Option<Double>> supplier = () -> 5.342;
```

Comment:
This should be declared as `Supplier<Double>`. Otherwise, the following lines won't compile. This is different from method `orElse( ... )`.